### ISSUE TITLE
The codeRequest with 'ok' status

### ISSUE THREAD

**Issue by pcabreus** [WHAnonymous/Chat-API#900]
I'm trying to send a codeRequest and that is the debug with registerTool.php:

```
Array
(
    [in] => 54000000
    [cc] => 53
    [id] =>vdr���\�t�;�g�:
    [lg] => es
    [lc] => CU
    [sim_mcc] => 368
    [sim_mnc] => 000
    [method] => sms
    [token] => f104b6ad96fd37523b8a08888a5b042d
)
stdClass Object
(
    [status] => ok
    [login] => 5354000000
    [pw] => **removed**
    [type] => existing
    [expiration] => 1463501383
    [kind] => free
    [price] => US$0.99
    [cost] => 0.99
    [currency] => USD
    [price_expiration] => 1441438696
)
```
and not some like that:
```
stdClass Object
(
    [status] => sent
    [length] => 6
    [method] => sms
    [retry_after] => 1805
)
```
Of course, the code never arrives and the passwork is not work. When I repeat the process do the same thing but the 'pw' change. 

Note: I changed the real phone number.

**Comment by 0xTryCatch** [comment]
If the password does not work you are doing something wrong

**Comment by mgp25** [comment]
@pcabreus you don't get the sms code because you have already set an identity to that number, you can remove your identity from the `wadata` folder and you'll get the SMS. Btw, the `pw` value should work

**Comment by pcabreus** [comment]
@mgp25 thank. I removed the id.<number>.dat from wadata folder and work fine. Thank but why do that? What mean this identity, I always need to remove it before send a codeRequest.

**Comment by pcabreus** [comment]
I'm making a php service for my symfony application to work with WhatsApp. I was using this library but I don't like so much its architecture style and not use namespace so I created my own class that use this one (github.com/pcabreus/WhatsAppApi). But in my last update, when I changed from mgp25/whastapi to whatsapp/chat-api I found that wadata folder to store caches. That is no a problem, the problem is the path. By default it's using a folder named /wadata inside the library. When I install the library by composer in vendor, I dont want that any library write/clean cache inside vendor directory. For cache files symfony for example propose an external folder far away from vendor, app/cache/wadata could be a good place to store my cache files. I need that could be possible to define the path where I want to store my cache files.

Be careful with my very bad English, sorry.
